src/detectors/B0ECAL/B0ECAL.cc

```diff
@@ -33,11 +33,12 @@ extern "C" {
           {"B0ECalRawHits"},
 #endif
           {
-            .eRes = {0.0 * sqrt(dd4hep::GeV), 0.02, 0.0 * dd4hep::GeV},
+                         // The stocastic term is set using light yield in PbOW4 of N_photons = 145.75 / GeV / mm, for 6x6 mm2 sensors with PDE=0.18 (a=1/sqrt(145.75*36*0.18))
+            .eRes = {0.0326 * sqrt(dd4hep::GeV), 0.02, 0.0 * dd4hep::GeV},
```

Comment:
I'm pretty sure you can remove the 2% term, it's probably was copied from Athena code for another calorimeter and has no meaning here.

Reply:
cc @Chao1009

Reply:
Also #1633

Reply:
thank you for your suggestion, I removed the 2% term. 

also cc'ing @FriederikeBock

Reply:
Yes, the smearing is better to be always zero unless there are some unknown sources for the resolution.